DirectProgramming/C++SYCL_FPGA/Tutorials/Features/ip_authoring_interfaces/component_interfaces_comparison/naive/README.md

```diff
@@ -0,0 +1,155 @@
+# Naive Design
+This simple vector add implementation is similar to the one presented in the [fpga_compile]((https://github.com/oneapi-src/oneAPI-samples/tree/master/DirectProgramming/C%2B%2BSYCL_FPGA/Tutorials/GettingStarted/fpga_compile)) code sample. It demonstrates a kernel that uses a register-mapped invocation interface, and a single shared memory-mapped host data interface. These are the default interfaces that are selected by the Intel® oneAPI DPC++/C++ Compiler, and therefore this is the only design that can support the full-system compilation flow.
+
+![](../assets/naive.svg)
+
+## Invocation Interface
+By default, an un-decorated oneAPI kernel will have all its control signals and arguments mapped into the IP component's control/status register (CSR).
+
+## Data Interface
+By default, the pointer arguments `a_in`, `b_in`, `c_out` and scalar argument `len` are passed through the IP component's CSR. The IP component accesses the data pointed to by the pointers through a single memory-mapped host interface.
```

Comment:
I want to highlight this ease-of-use feature we built where the data interfaces inherit the same style as the invocation interfaces unless customized to be otherwise.

```suggestion
Unless otherwise customized, the data interfaces inherit the same style as the invocation interfaces. Since by default, the invocation interface is a register-mapped interface, data interfaces also default to register-mapped interfaces.

Hence, in this design, the pointer arguments `a_in`, `b_in`, `c_out` and scalar argument `len` are passed through the IP component's CSR. The IP component accesses the data pointed to by the pointers through a single memory-mapped host interface.
```